### PR TITLE
cmd/script: usage without arguments shows preinit script

### PR DESCRIPTION
**Pull request by HBuczynski**
## Description

Command `script` without any arguments shows preinit script. Otherwise, the device, script's name and magic number is needed.

## Motivation and Context

Unification of commands.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (imxrt1064).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [x] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: cmds/script.c [hunk 1]
```diff
@@ -36,25 +36,15 @@ static int cmd_script(int argc, char *argv[])
 	char buff[SIZE_CMD_ARG_LINE];
 
 	if (argc == 1) {
-		log_error("\n%s: Arguments have to be defined", argv[0]);
+		lib_printf(CONSOLE_BOLD "\nPreinit script:");
+		lib_printf(CONSOLE_NORMAL "\n%s", (char *)script);
 		return -EINVAL;
```
Comment (agkaminski) [inline cmds/script.c:41]: Maybe we want to return 0 in this case

Comment (HBuczynski) [reply cmds/script.c]: Leavings from the previous condition. 
It should return EOK. Fixed.

**Review by agkaminski — DISMISSED** [review]

File: cmds/script.c [hunk 2]
```diff
@@ -24,7 +24,7 @@ extern char script[];
 
 static void cmd_scriptInfo(void)
 {
-	lib_printf("shows script, usage: script <preinit/name> [dev] [magic]");
+	lib_printf("shows script, usage: script <[dev] [name] [magic]>");
```
Comment (kemonats) [inline cmds/script.c:27]:
```suggestion
	lib_printf("shows script, usage: script [<dev> <name> <magic>]");
```
There may be no arguments or there must be 3.

Comment (HBuczynski) [reply cmds/script.c]: Done

**Pull request merged**